### ISSUE TITLE
Port to Python3

### ISSUE THREAD

**Issue by athivillon** [scumjr/yubikeyedup#7]
Hello,

I have successfully managed to port this code to python3.

```
athivillon@yoda:~/yubikeyedup-python3$ python3 tools/unittests.py
.........
----------------------------------------------------------------------
Ran 9 tests in 0.096s

OK
athivillon@yoda:~/yubikeyedup-python3$ python3 -V
Python 3.7.3

athivillon@yoda:~/yubikeyedup-python3$ git branch -a
  master
* python3
  remotes/origin/HEAD -> origin/master
  remotes/origin/master
```

If you are interested i can probably commit in a python3 branch.

The hardest job was to manage str != bytes in Crypto and HTTP Response.

I have also added a service file for starting it via systemd.

Let me know.

Thank you

**Comment by scumjr** [comment]
Nice, feel free to open a PR!

**Comment by athivillon** [comment]
Done !

I have also a branch working with YubiHSM2 and https://github.com/Yubico/python-yubihsm but for now it breaks compatibility in Database and will not start if python-yubihsm is not installed. I will try to unify this.

**Comment by scumjr** [comment]
I just merged the PR #8, feel free to open another one for YubiHSM2. Thanks!
